PowerShell/ScubaGear/Modules/Support/Support.psm1

```diff
@@ -28,14 +28,119 @@ function Copy-ScubaBaselineDocument {
         New-Item -ItemType Directory -Path $Destination | Out-Null
     }
 
-    @("teams", "exo", "defender", "aad", "powerplatform", "sharepoint") | ForEach-Object {
+    @("teams", "exo", "defender", "aad", "powerbi", "powerplatform", "sharepoint") | ForEach-Object {
```

Comment:
This appears to be a new service in the list of service, but per Crutch it is not a new service.  This requires comments to explain what is going on.